### PR TITLE
Tap trustpilot intial changes - Discover mode, Sync mode, bulk request.

### PR DESCRIPTION
**Pull request by sgandhi1311**
# Description of change
Code has been refactored to generate the catalog.json as per the stitch data standards (discover mode). Created separate file to handle the discover mode logic.
Modified sync mode logic to accept the command line arguments.
Modified consumers stream logic to handle the bulk request. 
Added circleci config.yml and changelog.md.
Unit tests and integration tests are yet to be done.

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+## 0.2.0
+* Fixed Discovery mode to add metadata and other missing fields in catalog.json
+* Fixed Sync mode to extract data
+* Changed consumer profile endpoint to extract data in bulk mode
```
Comment (KrisPersonal) [inline CHANGELOG.md]: This should be done as part of Bump version. Please remove this change for right now

File: setup.py [hunk 2]
```diff
@@ -3,16 +3,23 @@
 
 setup(
     name="tap-trustpilot",
-    version="0.1.0",
+    version="0.2.0",
```
Comment (KrisPersonal) [inline setup.py]: This again should be changed as part of Bump version

File: setup.py [hunk 3]
```diff
@@ -3,16 +3,23 @@
...
     author="Stitch",
     url="http://singer.io",
     classifiers=["Programming Language :: Python :: 3 :: Only"],
     py_modules=["tap_trustpilot"],
     install_requires=[
-        "singer-python>=5.0.12",
+        "singer-python",
         "requests",
     ],
+    extras_require={
```
Comment (KrisPersonal) [inline setup.py:17]: Please add comments to the code

**Review by KrisPersonal — COMMENTED** [review]
Do we have unit test cases?

File: setup.py (hunk 2 above)
Comment (kethan1122) [reply setup.py]: done

File: setup.py (hunk 3 above)
Comment (kethan1122) [reply setup.py:17]: done

File: CHANGELOG.md (hunk 1 above)
Comment (kethan1122) [reply CHANGELOG.md]: done

**Review by RushiT0122 — APPROVED** [review]

**Pull request merged**